### PR TITLE
PLT-857: Support `attachments` for Incoming Webhooks

### PR DESCRIPTION
**Pull request by florianorben**
This adds attachments as described in https://api.slack.com/docs/attachments
["Automatic unfurling"](https://api.slack.com/docs/unfurling) is not implemented - my guess is, slack has a bunch of rules for different websites, telling the application what to crawl/fetch.

Addionally, attachments are currently not visible in RHS comment section, I was not quite sure on how to display it - since there is very little space available, do you have any idea - probably leave of any images in the attachments etc.?
Thought I'd already submit the PR anyway since I won't be able to work on it this saturday - perhaps someone can already have a look at it.

Just tell me what/if there's any work to be done!

*Test webhook body, from an up2date gitlab version (if you need one for testing):*
````javascript
{
    "text": "Florian pushed to branch \u003chttp://srv:3000/foo/bar/commits/feature/update-33390|feature/update-33390\u003e of \u003chttp://srv:3000/foo/bar|Foo/Bar\u003e (\u003chttp://srv:3000/foo/bar/compare/5829dfe201a9dbff15729ab99cd33b64c8e11236...36028fd8d43c4434a048f66597588d46fd958733|Compare changes\u003e)",
    "username": "gitlab",
    "channel": "town-square",
    "attachments": [
        {
            "text": "\u003chttp://srv:3000/foo/bar/commit/36028fd8d43c4434a048f66597588d46fd958733|36028fd8\u003e: Some very important fix\n - flo\n\u003chttp://srv:3000/foo/bar/commit/d1291f692b9944535bb569f6841d2972cec4d8ff|d1291f69\u003e: Another important fix\n - mr x\n\u003chttp://srv:3000/foo/bar/commit/5829dfe201a9dbff15729ab99cd33b64c8e11236|5829dfe2\u003e: Some changes\n - someone",
            "color": "#345"
        }
    ]
}
````
 
bildschirmfoto vom 2015-10-31 04 59 25

### CODE REVIEW

File: api/post.go [hunk 1]
```diff
@@ -212,6 +212,7 @@ func handlePostEventsAndForget(c *Context, post *model.Post, triggerWebhooks boo
 		}
 
 		if triggerWebhooks {
+			fmt.Println("triggerWebhooks", post)
```
Comment (jwilander) [inline api/post.go]: I assume this was for debugging? Should probably be removed :)

Comment (florianorben) [inline api/post.go]: Ooops, yep :) I'll get rid of that

Comment (florianorben) [inline api/post.go]: @jwilander Debug line removed :) + rebased on current master

**Comment by jwilander** [comment]
We're talking with @florianorben about some changes, so closing for now

**Pull request closed without merging**